### PR TITLE
Fix logo on PyPI page

### LINKED ISSUE #3416: Get rid of GitHub magic in README [Quick Win]
@marrrcin reported that he saw the README like this

image

and it's because of a bad combination of the magic we're using to detect light/dark theme

https://github.com/kedro-org/kedro/blob/8257d55ec96abbc1ce18ae229da8c773c60bfc1b/README.md?plain=1#L1-L2

and how GitHub assumes night = dark

image

This has created other problems in the past https://github.com/kedro-org/kedro/pull/2961#issuecomment-1688542270

Given that this is broken on PyPI and half-broken on GitHub, I'd say let's get rid of this.

A strawman proposal would be something like

image

But better to have this properly designed.

### PR DESCRIPTION
**Pull request by ravi-kumar-pilla**
## Description

Resolves #3416 

## Development notes

Updated README according to the GitHub recommendation for [specifying-the-theme-an-image-is-shown-to](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) 

**Testing -**

**GitHub Light mode:**

Screenshot 2024-08-26 at 7 17 33 PM

**GitHub Dark mode:**

Screenshot 2024-08-26 at 7 16 58 PM

**PyPi Light/Dark mode:**

Screenshot 2024-08-26 at 7 17 47 PM

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [x] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -3,7 +3,10 @@
 ## Major features and improvements
 ## Bug fixes and other changes
 ## Breaking changes to the API
+
 ## Documentation changes
+* Add theme support for Kedro logo in README.
```
Comment (astrojuanlu) [inline RELEASE.md:8]:
```suggestion
* Fix logo on PyPI page
```

?

**Review by astrojuanlu — APPROVED** [review]
Finally ❤️ Thanks @ravi-kumar-pilla !

**Review by merelcht — APPROVED** [review]
Thanks @ravi-kumar-pilla ⭐

File: RELEASE.md (hunk 1 above)
Comment (ravi-kumar-pilla) [reply RELEASE.md]: Sure

**Pull request merged**